Userland/Services/RequestServer/main.cpp

```diff
@@ -20,15 +20,15 @@
 ErrorOr<int> serenity_main(Main::Arguments)
 {
     TRY(Core::System::pledge("stdio inet accept unix rpath sendfd recvfd sigaction"));
-
     signal(SIGINFO, [](int) { RequestServer::ConnectionCache::dump_jobs(); });
+    TRY(Core::System::pledge("stdio inet accept unix rpath sendfd recvfd"));
 
     // Ensure the certificates are read out here.
     [[maybe_unused]] auto& certs = DefaultRootCACertificates::the();
+    TRY(Core::System::pledge("stdio inet accept unix sendfd recvfd"));
 
     Core::EventLoop event_loop;
     // FIXME: Establish a connection to LookupServer and then drop "unix"?
```

Comment:
Please move this comment along with the pledge it's about.

Reply:
Can do, but dropping `rpath` earlier seems (to me) to be independent of the desire to drop `unix` later on in the program, where the comment is?

Reply:
Ah, sorry, I misread the source here, I thought that comment was about the pledge.
my bad :P